docs/sys-utilities/gpactivatestandby.md

```diff
@@ -0,0 +1,96 @@
+
+If the primary coordinator fails, the log replication process is shutdown, and the standby coordinator can be activated in its place by using the `gpactivatestandby` utility. Upon activation of the standby coordinator, the replicated logs are used to reconstruct the state of the Cloudberry coordinator host at the time of the last successfully committed transaction.
+
+In order to use `gpactivatestandby` to activate a new primary coordinator host, the coordinator host that was previously serving as the primary coordinator cannot be running. The utility checks for a `postmaster.pid` file in the data directory of the deactivated coordinator host, and if it finds it there, it will assume the old coordinator host is still active. In some cases, you may need to remove the `postmaster.pid` file from the deactivated coordinator host data directory before running `gpactivatestandby` (for example, if the deactivated coordinator host process was terminated unexpectedly).
+
+After activating a standby coordinator, run `ANALYZE` to update the database query statistics. For example:
+
+```shell
+psql <dbname> -c 'ANALYZE;'
+```
+
+After you activate the standby coordinator as the primary coordinator, the Cloudberry Database system no longer has a standby coordinator configured. You might want to specify another host to be the new standby with the [gpinitstandby](/docs/db-utilities/db-util-gpinitstandby.md) utility.
+
+## Options
+
+**`-a (do not prompt)`**
+
+Do not prompt the user for confirmation.
+
+**`-d standby_coordinator_datadir`**
+
+The absolute path of the data directory for the coordinator host you are activating.
+
+If this option is not specified, `gpactivatestandby` uses the value of the `COORDINATOR_DATA_DIRECTORY` environment variable setting on the coordinator host you are activating. If this option is specified, it overrides any setting of `COORDINATOR_DATA_DIRECTORY`.
+
+If a directory cannot be determined, the utility returns an error.
+
+**`-f (force activation)`**
+
+Use this option to force activation of the backup coordinator host. Use this option only if you are sure that the standby and primary coordinator hosts are consistent.
+
+**`-l logfile_directory`**
+
+The directory to write the log file. Defaults to `~/gpAdminLogs`.
+
+**`-q (no screen output)`**
+
+Run in quiet mode. Command output is not displayed on the screen, but is still written to the log file.
+
+**`-v (show utility version)`**
+
+Displays the version, status, last updated date, and check sum of this utility.
+
+**`-? | -h | --help (help)`**
+
+Displays the online help.
+
+## Example
+
+Activate the standby coordinator host and make it the active coordinator instance for a Cloudberry Database system (run from backup coordinator host you are activating):
+
+```shell
+gpactivatestandby -d /gpdata
+```
+
+## See also
+
+[gpinitsystem](/docs/db-utilities/db-util-gpinitsystem.md), [gpinitstandby](/docs/db-utilities/db-util-gpinitstandby.md)
```

Comment:
the dir name has changed from `db-utilities` to `sys-utilities`.

Reply:
Thanks for the comment. Updated the value in links.